manuscript/revision/response_to_editor.tex

```diff
@@ -0,0 +1,225 @@
+\documentclass{letter}
+\usepackage{xcolor}
+\usepackage{hyperref}
+\signature{Savita Karthikeyan}
+
+\address{Big Data Institute\\University of Oxford\\UK}
+\begin{document}
+
+\begin{letter}{}
+
+\opening{Dear Alison,}
+
+We are delighted that the reviewers have found our manuscript suitable for publication 
+in Bionformatics. We thank you, the editors and all the reviewers for the careful 
+consideration of our manuscript, and value the kind words and constructive feedback 
+provided on our work. We have now revised the manuscript based on the reviewers' 
+suggestions. Below is our point-by-point response to the 
+reviewers' comments in blue. We have incorporated as many suggestions as possible, 
```

Comment:
I would delete the last sentence "We have incor"... It goes without saying that we incorporate the suggestions, and it's just raising the expectation that we're pushing back on something major to say "where unable to do so"